source/includes/overview/_quickstart.md

```diff
@@ -5,31 +5,45 @@ Zenodo, and will be using Python together with the
 [Requests](http://www.python-requests.org/en/latest/user/install/) package.
 
 ```terminal
-$ pip install requests
+# Install 'requests' module for python
+pip install requests
```

Comment:
I agree on being consistent actually throughout and removing the `$` or in Python the `>>>`. Since our Slate version is too old, we cannot use [any copy-to-clipboard magic](https://github.com/slatedocs/slate/pull/1258).

The "issue" is how to represent output in examples (I have a preference for the second):

```python
>>> print("foobar")
foobar
```

vs. 

```python
print("foobar")
# foobar
```

Reply:
The idea is to make copy / paste easier. We can add the `.css` as @ppanero, what do you think? I can take care of that in this PR.
 
I agree with @slint that we should be consistent and I also prefer removing the `>>>` from the snippets. I would suggest that, instead of:

``` python
print("foobar")
# foobar
```

we have something like:

---

```python
print("foobar")
```

The expected output is the following: 

```python
foobar
```
---

Got this idea from [google style guide](https://developers.google.com/style/code-syntax)

Reply:
I agree with @slint that we should be consistent and I also prefer removing the `>>>` from the snippets. I would suggest that, instead of:

``` python
print("foobar")
# foobar
```

we use something like:

---

```python
''' This does something '''
print("foobar")
# foobar
```
---

With this suggestion, triple quotes (`'''`) refers to code doc while hashtash (`#`) is solely used to display the code's output. WDYT? 

There was a nice idea from [google style guide](https://developers.google.com/style/code-syntax) about splitting code from output in different blocks, however it does not look good (content looked "splitted" between left and right).